meteor-app/imports/ui/filter-value.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Make sure the filter value is correct.
+ */
+
+export const checkFilterValue = (value, min, max) => {
+  let newValue = parseInt(value, 10);
+  newValue = isNaN(newValue) ? min : newValue;
+  newValue = Math.max(min, newValue);
+  newValue = Math.min(newValue, max);
+  return newValue;
+}
```

Comment:
When I said create a helper function I meant creating `/imports/ui/helpers` which could be `/imports/ui/helpers.js` or `/imports/ui/helpers/index.js`.

`filter-value.js` is not a name that belongs here.